src/main/src/test/java/org/geoserver/test/GeoServerSystemTestSupport.java

```diff
@@ -2013,6 +2014,19 @@ protected int countNonBlankPixels(String testName, BufferedImage image, Color bg
         return pixelsDiffer;
     }
 
+    // TODO: consider moving to GeoServerSystemTestSupport
```

Comment:
isn't indeed moved?

Reply:
Oh LOL yes, at first I was not sure about moving it and then I did and forgot the comment.